powerlevel9k.zsh-theme

```diff
@@ -557,6 +557,15 @@ prompt_nvm() {
   $1_prompt_segment "$0" "$2" "green" "011" "${node_version:1}" 'NODE_ICON'
 }
 
+# NodeEnv Prompt
+prompt_nodeenv() {
+  local nodeenv_path="$NODE_VIRTUAL_ENV"
+  if [[ -n "$nodeenv_path" && "$NODE_VIRTUAL_ENV_DISABLE_PROMPT" != true ]]; then
```

Comment:
Its related to [nodeenv](https://github.com/ekalinin/nodeenv/blob/2cd7de2005c4e33cc324ecd2cf84c995ebcd2d19/nodeenv.py#L929
). 
I will do the necessary update to README shortly.